.bazelrc

```diff
@@ -22,6 +22,22 @@ build:ci --features=layering_check
 build --java_language_version=8
 build --tool_java_language_version=8
 
+# Android launcher build config
+build:android_binary --crosstool_top=@androidndk//:toolchain
+build:android_binary --incompatible_enable_cc_toolchain_resolution=false
```

Comment:
Could you try to remove this line and instead add `--incompatible_enable_android_toolchain_resolution`? All of these flags are [planned to be flipped in Bazel 7](https://github.com/bazelbuild/bazel/issues/7260), so I would prefer to keep using them.